src/modules/major.js

```diff
@@ -0,0 +1,24 @@
+import { reducerUtils,createPromiseThunk, handleAsyncActions } from "../lib/asyncUtils";
+import { infoAPI } from "../api";
+
+const GET_MAJOR_LIST="GET_MAJOR_LIST";
+const GET_MAJOR_LIST_SUCCESS="GET_MAJOR_LIST_SUCCESS";
+const GET_MAJOR_LIST_ERROR="GET_MAJOR_LIST_ERROR";
+
+export const getMajorList = createPromiseThunk(GET_MAJOR_LIST, infoAPI.getStudentNumberList);
+
+const initialState = {
+  studentNumberList:reducerUtils.initial([]),
```

Comment:
아래처럼 안 쓴 이유는 뭔가요? 궁금
```suggestion
  studentNumberList: [],
```

Reply:
기존 코드에서 asyncUtils파일에 초기 값을 관리하는 코드를 다른 redux파일에서 사용하길래 저도 통일하는 느낌으로 사용해봤습니다 ㅎ